### PR TITLE
Add multiple poses

### PR DESCRIPTION
**Pull request by apayne97**
## Description
For some reason the `POSIT` algorithm doesn't seem to be performing sufficient conformer enumeration, so this PR adds the ability to manually enumerate conformations using `Omega` and pass those conformers to `POSIT`.

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [x] Modify docking function to be able to perform `Omega` conformer enumeration
  - [x] Modify docking script to have new CLI args for the docking function args

## Status
- [x] Ready to go

### CODE REVIEW

File: asapdiscovery/docking/docking.py [hunk 1]
```diff
@@ -212,19 +244,23 @@ def run_docking_oe(
...
-        pose_scorer.Initialize(du)
-        chemgauss_score = pose_scorer.ScoreLigand(posed_mol)
+    if (
+        ret_code == oedocking.OEDockingReturnCode_Success
+        and dock_sys == "posit"
+    ):
+        posed_mols = []
+        posit_probs = []
+        posit_methods = []
+        for r in pose_res.GetSinglePoseResults():
```
Comment (JenkeScheen) [inline asapdiscovery/docking/docking.py:258]: best to use a more verbose variable name than `r` for readability

File: asapdiscovery/docking/mcs.py [hunk 2]
```diff
@@ -287,7 +287,7 @@ def rank_structures_rdkit(
             d.FinishDrawing()
             d.WriteDrawingText(f"{out_fn}_{search_ids[mol_idx]}_{i}.png")
 
-    Draw.MolToFile(exp_mol, f"{out_fn}.png")
+        Draw.MolToFile(exp_mol, f"{out_fn}.png")
```
Comment (JenkeScheen) [inline asapdiscovery/docking/mcs.py:290]: could we make this optional? I'm not sure what the `.png`s are used for and I end up just deleting them manually.

**Review by JenkeScheen — CHANGES_REQUESTED** [review]
looks great!

File: asapdiscovery/docking/docking.py (hunk 1 above)
Comment (kaminow) [reply asapdiscovery/docking/docking.py]: good point, fixed in [0ac9c57](https://github.com/choderalab/covid-moonshot-ml/pull/93/commits/0ac9c57fd745f515a5333d27673081821e5bc707)

File: asapdiscovery/docking/mcs.py (hunk 2 above)
Comment (kaminow) [reply asapdiscovery/docking/mcs.py:290]: as of [64ad66f](https://github.com/choderalab/covid-moonshot-ml/pull/93/commits/64ad66ffc4c5dc02167a3b3c09f731a4172c94b3) the script won't draw the molecules if you set `-n_draw 0`

**Review by JenkeScheen — APPROVED** [review]

**Pull request merged**